src/ilcdlib/soda4lca/api_client.py

```diff
@@ -137,19 +144,20 @@ def download_epd_document(self, process_uuid: str, version: str | None = None) -
             raise e
 
     def search_processes(
-        self, offset: int = 0, page_size: int = 100, lang: str | None = None, **other_params
+        self, offset: int = 0, page_size: int = 100, lang: str | None = None, use_xml=True, **other_params
     ) -> ProcessSearchResponse:
         """
         Filter processes by various criteria.
 
         :param offset: offset from the beginning of the dataset
         :param page_size: size of the page
         :param lang: language code (2 letters)
+        :param use_xml: if True, response will be handled as XML
```

Comment:
Why `use_xml` is method argument and a property of a "Client"? The Client is mixing xml and json?

Reply:
Removed json